src/BlazorUI/Bit.BlazorUI/Components/Buttons/ActionButton/BitActionButton.scss

```
@@ -125,15 +127,15 @@
             left: 0;
             right: 0;
             bottom: 0;
-            height: 1px;
+            height: $shp-border-width;
             content: "";
             position: absolute;
             background-color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)